<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArticleForm</class>
 <widget class="QWidget" name="ArticleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Справочники</string>
  </property>
  <widget class="QGroupBox" name="gB">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>610</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>Справочник-федеральный розыск</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выборка  базы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Отпечатки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ввод записи</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewfed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>581</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия Имя Отчество</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="N_lineEdit_fed">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fed">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>531</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Статьи наказания</string>
    </property>
    <widget class="QComboBox" name="artnum_comboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="art_comboBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Номер</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Содержание</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Срок</string>
     </property>
    </widget>
    <widget class="QComboBox" name="srok_comboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="res_comboBox">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Срок рецедива</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Печать</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>610</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>Справочник- статьи наказания</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выборка  базы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Содержание статьи</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ввод записи</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>321</y>
      <width>591</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер статьи </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="A_textEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="N_lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Срок наказания</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Срок наказания -рецедив</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="res_spinBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>190</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="nak_spinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50000</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>600</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>1151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Справочники</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
